spec/cases/points_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe DummyMetrics do
+  before do
+    stub_request(:get, "http://localhost:8086/query")
+      .with(
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.